src/components/NavigationMailbox.vue

```diff
@@ -262,6 +271,23 @@ export default {
 		isSubscribed() {
 			return this.mailbox.attributes && this.mailbox.attributes.includes('\\subscribed')
 		},
+		isDroppableSpecialMailbox() {
+			return ['inbox', 'junk', 'trash'].includes(this.mailbox.specialRole)
```

Comment:
the flags aren't always accurate. we should use the account preferences from https://github.com/nextcloud/mail/issues/2035, like `account.sentMailboxId`

Reply:
I turned it around into a blacklist, using `sentMailboxId` and `draftsMailboxId`. Additionally, I am checking if `this.filter === 'starred'` to exclude the "Favorites"  mailbox as well (the move doesn't list the favorites either).